README.md

```diff
@@ -0,0 +1,18 @@
+## SportZone
+Software para asesoramiento deportivo
+
+
+## Problema a solucionar
+Muchas veces hemos querido comenzar a llevar una vida más saludable iniciandonos en algún deporte pero no sabemos bien como empezar bien por la falta de experiencia en este o por la ausencia de entrenadores de ese deporte en nuestra zona. 
+
+
+## Solución propuesta
+SportZone es un proyecto que tiene como fin que sus usuarios puedan iniciarse en el deporte que deseen con planes personalizados acorde con los parámetros que se indiquen como "deporte seleccionado", "tiempo disponible semanal", "estado físico actual", ...
+El sistema analizará los parámetros introducidos por el usuario y a partir de ellos generará un planning personalizado que incluirá sesiones de entrenamiento, recomendaciones de suplementación antes, durante y después las sesiones, y material deportivo necesario.
+
+
+## Lógica de negocio
```

Comment:
Por favor, consulta lo que se ha explicado sobre lógica de negocio, qué es y por qué es necesaria.

Reply:
> Por favor, consulte lo que se ha explicado sobre lógica de negocio, qué es y por qué es necesario.

Lo revisaré y modificaré, pero ¿cree que la idea del proyecto de un sistema que cree plannigs de entrenamiento de acuerdo a unas pautas que especificamente el usuario es correcta o cree que debo buscar otro tipo de proyecto?

Reply:
En todo caso, como te digo abajo, "un sistema que haga" no es lo que se pide, sino que entendáis un problema, no que propongáis una solución (que no lo es) a un problema inexistente o indeterminado.